dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss

```diff
@@ -3043,6 +3043,17 @@ div.filesTable {
 	padding: 0 4px;
 }
 
+.datasetfield-value {
+	p {
+		margin-bottom: 0;
```

Comment:
I don't think that we really want to get rid of margin on `p`. Some time ago we had an issue #1753 and decided that rich editor (description field) styling should be as close as possible to the style of display

Reply:
Okay - didn't realise that, my bad. The reason I've removed the bottom margin was to keep the distance between text and borders (content separators) even. Having that in mind, I've changed it so that an additional top margin is added instead to every element except for first.